src/polyswarm_api/__main__.py

```diff
@@ -332,29 +269,29 @@ def lookup(ctx, uuid, uuid_file):
 
 @click.option('-r', '--hash-file', help='File of hashes, one per line.', type=click.File('r'))
 @click.option('-m', '--metadata', is_flag=True, default=False, help='Save file metadata into associated JSON file')
-@click.option('--hash-type', help='Hash type to search [sha256|sha1|md5], default=sha256', default='sha256')
+@click.option('--hash-type', help='Hash type to search [default:sha256|sha1|md5]', default=None)
 @click.argument('hash', 'hash', nargs=-1, callback=validate_hash)
 @click.argument('destination', 'destination', nargs=1, type=click.Path(file_okay=False))
 @polyswarm.command('download', short_help='download file(s)')
 @click.pass_context
 def download(ctx, metadata, hash_file, hash_type, hash, destination):
-    if not os.path.exists(destination):
-        os.makedirs(destination)
-
+    """
```

Comment:
When adding docstrings, please document the parameters as well. There's a backlog of these but as we are doing it, good to do it right.

Reply:
yep, it's missing in all the funcs around.

Reply:
we should document all at once in next PR.